src/main/scala/io/iohk/ethereum/db/components/StoragesComponent.scala

```diff
@@ -19,7 +19,7 @@ trait StoragesComponent {
 
     val mptNodeStorage: MptNodeStorage
 
-    val nodeStorage: NodeStorage //FIXME This storage is similar to MPTNodesStorage but
+    val nodeStorage: NodeStorage //FIXME This storage is similar to MPTNodesStorage, should we keep only one? [EC-147]
```

Comment:
Yes...maybe we need to remove the "configurable hash" on `Mpt` . I think the unification and storage removal is a good idea but it's a task that will take some hours. I would create a ticket in order to address this in the future tho.